baggers.js

```diff
@@ -1,6 +1,47 @@
 // List of baggers...
 var data = {
   speakers: [
+    {
+      since: '2020-26-02',
+      name: 'Thomas Tourlourat',
+      bio:
+        'Lead-dev chez Copines de Voyage, ex @Konbini, ex @Freelance',
+      picture:
+        'https://pbs.twimg.com/profile_images/714783052052971520/WaNkikC9_400x400.jpg',
+      websites: [
+        {
+          name: 'Github',
+          url: 'https://github.com/armetiz',
+        },
+        {
+          name: 'Twitter',
+          url: 'https://twitter.com/armetiz',
+        },
+        {
+          name: 'Malt',
+          url: 'https://www.malt.fr/profile/thomastourlourat',
+        },
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/tourlourat/',
+        }
+      ],
+      location: "Les Sables-d'Olonne",
```

Comment:
Great to add a new location. We need then to also add `Les Sables-d'Olonne` as a new city in the cities dataset.

Please check: https://github.com/brownbaglunch/bblfr_data/#cities-dataset